src/app/shared/cookies/klaro-configuration.ts

```diff
@@ -31,10 +31,7 @@ export const klaroConfiguration: any = {
   */
   hideLearnMore: false,
 
-  /*
-  Override cookies modal styles according to customer TUL requirements.
-   */
-  stylePrefix: 'klaro customer-cookies',
```

Comment:
Why? It could be used in the future, now that CSS is empty

Reply:
Comment makes it sound like it is only for TUL.